### ISSUE TITLE
Tensorflow js returns NAN on large data set 

### ISSUE THREAD

**Issue by quickrfq** [tensorflow/tensorflow#18493]
<html>
--
  | <head>
  | 
  | <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@0.9.0"> </script>
  |  
  | 
  | <script>
  | 
  | 
  | 
  | 
  | 
  | 
  | const model = tf.sequential();
  | model.add(tf.layers.dense({units: 1, inputShape: [1]}));
  | model.compile({loss: 'meanSquaredError', optimizer: 'sgd'});
  | const xs = tf.tensor2d([1, 2, 3, 4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24], [24, 1]);
  | const ys = tf.tensor2d([3, 4, 5, 6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26], [24, 1]);
  | model.fit(xs, ys).then(() => {
  | model.predict(tf.tensor2d([11], [1, 1])).print();
  | });
  | </script>
  | </head>
  |  
  | <body>
  | </body>
  | </html>

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
